lib/src/cmaf/ham/mapper/hls/mapHlsToHam.ts

```diff
@@ -121,13 +158,13 @@ function mapHlsToHam(manifest: Manifest) {
 		};
 		const map = parsedHlsManifest.segments[0]?.map;
 		const byterange = map?.byterange;
-		const uri = map?.uri;
+		const codec = CODECS.split(',').at(0);
```

Comment:
Maybe separator should be included as a constants also.
On another hand, is it always placed in that position video CODECS?